notebook/static-src/auth/js/logoutmain.js

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-define(['base/js/namespace', 'base/js/page'], function(IPython, page) {
-    function logout_main() {
+    var IPython = require('base/js/namespace');
+    var page = require('base/js/page');
+
+    module.exports = function logoutMain() {
```

Comment:
Just a note that since we were returning a function it's cool that the name changed here to the JavaScriptism of CamelCase.